### ISSUE TITLE
Incorrect LaTeX syntax results in wrong line skip

### ISSUE THREAD

**Issue by samcarter** [nmfs-opensci/quarto_titlepages#34]
System settings:

- Quarto v1.3.49
- macOS 11.7.2
- Texlive 2022

### Description

A simple test file like

```
---
title: "Some Very Long Title Which Spans Multiple Lines "  
format:  
  titlepage-pdf:  
    titlepage-theme:   
      title-fontstyle: Huge
---  

test
```

will result in the following latex syntax for the title:

```
{{\Huge{\nohyphens{Some Very Long Title Which Spans Multiple
Lines}}}\par
}%
```

There are a couple of problems with this syntax. The most important one is that the paragraph is ended at the wrong position. If you insert `\par` only after you switched back from the custom font size, then the text will be set with a wrong baselineskip (the one from the normal font around it and not with your title font). Try the following two lines to see the difference:

```
{{\Huge{\nohyphens{Some Very Long Title Which Spans Multiple
Lines}}}\par
}%

{{\Huge{\nohyphens{Some Very Long Title Which Spans Multiple
Lines}}\par}
}%
```
<img width="540" alt="Screenshot 2023-02-08 at 10 45 56" src="https://user-images.githubusercontent.com/43832342/217494165-656146e3-8a76-4c1b-91ce-240cf2c2a2de.png">

Another potential problem: Depending on what comes after the title, you might get an additional space from the unprotected line ending after `\par}`

Not a problem as such, just unnecessary:

- the outermost `{...}` is unnecessary
- font size commands like `\Huge` etc are switches and don't take an argument. Instead of writing `\Huge{...}` it should normally be `\Huge ...`

**Comment by eeholmes** [comment]
Thanks for bringing this up. I am not able to replicate your image. Mine looks ok with the title broken where expected.

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/2545978/226378922-695e8d44-a3f9-42e9-8d5d-e76301054670.png">

Re font size commands, I don't know what font styles the user will use and `\whatevertheyputin{...}` seemed to be a fairly robust solution. However in general the font specs commands, like `\Huge`, have been a bit flakey. Using `title-fontsize` which takes a different approach to setting font size has been more robust.

My system settings and what I compiled with:

- Quarto v1.2.335
- nmfs-opensci/titlepage v3.3.8  
- macOS 11.6.2
- XeTeX 3.141592653-2.6-0.999994 (TeX Live 2022)

I'll try updating my quarto to the latest version and see if that changes things.

**Comment by samcarter** [comment]
> Thanks for bringing this up. I am not able to replicate your image. Mine looks ok with the title broken where expected.

The break point is fine, but also evident in your image, the spacing between the lines is wrong. The `y` and the `L` nearly touch each other.

**Comment by eeholmes** [comment]
EEH: to do for this issue

- [ ] Try changing the placement of the \par 
- [ ] Try adding % after the \par to get rid of the line feed
- [ ] The issue with `\Huge ...` being the right syntax won't fix since I don't know what the user will pass in for fontstyle and some need \foo{} syntax. Anyhow `\Huge{}` seems to work 'ok'.

```
{{\Huge{\nohyphens{Some Very Long Title Which Spans Multiple
Lines}}}\par
}%

{{\Huge{\nohyphens{Some Very Long Title Which Spans Multiple
Lines}}\par}
}%
```
